lang/cs/Org.Apache.REEF.Tests/Functional/Failure/EvaluatorWithRunningTaskDelayedPoisonTest.cs

```diff
@@ -107,13 +94,13 @@ public void OnNext(IActiveContext value)
                     .BindIntNamedParam<CrashMinDelay>("100")
```

Comment:
You can create a configuration module for Poison Configuration. Later whenever you need to get a config for poison, get the Configuration Module then set the values needed.